nemo/src/io/lexer.rs

```diff
@@ -166,6 +222,7 @@ impl std::fmt::Display for TokenKind {
             TokenKind::PrefixIdent => write!(f, "PrefixIdent"),
             TokenKind::Illegal => write!(f, "Illegal"),
             TokenKind::Eof => write!(f, "Eof"),
+            TokenKind::Error => write!(f, "\x1b[1;31mError\x1b[0m"),
```

Comment:
Please don't use raw escape sequences, and also don't colour unconditionally. In `nemo-cli` we already use [`colored::Colorize`](https://docs.rs/colored/latest/colored/trait.Colorize.html) for this, which handles, e.g., the `NO_COLOUR` environment variable and terminal capabilities.